crypto/src/lib.rs

```diff
@@ -206,7 +221,7 @@ pub trait Crypto {
         algs: AlgLen,
         measurement: &Digest,
         info: &[u8],
-    ) -> Result<Self::Cdi, CryptoError>;
+    ) -> Result<[u8; MAX_EXPORTED_CDI_SIZE], CryptoError>;
```

Comment:
Note from discussion, use `ExportedCdiHandle` for these types (here & derive_key_pair_exported)